openeo-geotrellis/src/main/scala/org/openeo/geotrellis/CustomizableHttpRangeReader.scala

```diff
@@ -1,17 +1,27 @@
 package org.openeo.geotrellis
 
 import geotrellis.util.RangeReader
+import org.slf4j.{Logger, LoggerFactory}
 import scalaj.http.HttpRequest
 
 import scala.util.Try
 
+
 // copied from geotrellis.util.HttpRangeReader as that one is not extensible
 class CustomizableHttpRangeReader(request: HttpRequest, useHeadRequest: Boolean) extends RangeReader {
+  private implicit val logger: Logger = LoggerFactory.getLogger(this.getClass)
```

Comment:
Unnecessary to create a new `Logger` instance every time. Create a `CustomizableHttpRangeReader` companion object with a `logger` field:

```suggestion
  private implicit val logger: Logger = LoggerFactory.getLogger(classOf[CustomizableHttpRangeReader])
```